securedrop_client/export.py

```diff
@@ -152,7 +152,14 @@ def _export_archive(cls, archive_path: str) -> ExportStatus:
                 ],
                 stderr=subprocess.STDOUT,
             )
-            return ExportStatus(output.decode("utf-8").strip())
+            result = output.decode("utf-8").strip()
+
+            # No status is returned for successful `disk`, `printer-test`, and `print` calls.
+            # This will change in a future release of sd-export.
+            if result:
+                return ExportStatus(result)
+            else:
+                return None
```

Comment:
I was wondering if there could be some value of `output`, different from `b''` that would lead to returning `None` (success).

Because `result` is always a string, and strings only evaluate to `False` when their length is 0, this is equivalent to `if result == ""; else` which is what we want.

However, given we `strip()` the decoded output, it means that any amount of white space would actually be considered as a success status. That is not new to this PR, but I think it's worth noting and maybe constraining?

Reply:
Since it's not a regression and since the new sd-export will always return a success status and this will all be deprecated, I would say it's not worth fixing it in this context, it makes more sense to push ahead with the new export API that exposes things like `SUCCESS_EXPORT`, `SUCCESS_PRINT_PREFLIGHT`, etc.